MobileCenterDistribute/MobileCenterDistributeTests/MSDistributeTests.m

```diff
@@ -724,18 +740,12 @@ - (void)testDontShowConfirmationAlertIfNoMandatoryReleaseWhileNoNetwork {
     [expection fulfill];
   });
   [self waitForExpectationsWithTimeout:1
-                               handler:^(__attribute__((unused)) NSError *error) {
-
-                                 /*
-                                  * Then
-                                  */
-                                 OCMReject(
```

Comment:
`OCMReject` needs to call _before_ testing method.
Also verify call needed when:
> In fail-fast mode an exception might not cause the test to fail. This can happen when the call stack for the method does not end in the test. Fail fast exceptions will be re-thrown when `OCMVerifyAll` is called.